client/index.html

```diff
@@ -468,6 +474,64 @@ <h1 class="title">Help</h1>
 							</div>
 						</div>
 
+						<h2>Autocompletion</h2>
+
+						<p>
+							The Lounge offers automatic completion of nicknames, channels, <kbd>/</kbd>-commands, emoji, and text colors.
+							Type one of the characters below to get a menu of suggestions, and filter the list by typing part of the channel
+							name, nickname, emoji, command, or color you want. Use the up and down arrow keys to highlight a menu item.
+							Insert the highlighted suggestion by pressing <kbd>Tab</kbd> or <kbd>Enter</kbd>, or click the desired item.
+						</p>
+						<p>
+							Autocompletion can be disabled in settings. (Typing part of a nickname and pressing <kbd>Tab</kbd> will always
+							complete it, regardless of setting state.)
+						</p>
+
+						<div class="help-item">
+							<div class="subject">
+								<code>@</code>
+							</div>
+							<div class="description">
+								<p>Nickname</p>
+							</div>
+						</div>
+
+						<div class="help-item">
+							<div class="subject">
+								<code>#</code>
+							</div>
+							<div class="description">
+								<p>Channel</p>
+							</div>
+						</div>
+
+						<div class="help-item">
+							<div class="subject">
+								<code>/</code>
+							</div>
+							<div class="description">
+								<p>Commands (see list of commands below)</p>
+							</div>
+						</div>
+
+						<div class="help-item">
+							<div class="subject">
+								<code>:</code>
+							</div>
+							<div class="description">
+								<p>Emoji (note: requires two search characters, to avoid conflicting with common emoticons like <code>:)</code>)</p>
+							</div>
+						</div>
+
+						<div class="help-item">
+							<div class="subject">
+								<kbd>Ctrl</kbd> + <kbd>K</kbd>
```

Comment:
Make this `<kbd class="key-all">Ctrl</kbd><kbd class="key-apple">⌘</kbd>`

Reply:
Fixup pushed.